test/e2e-common.sh

```diff
@@ -42,7 +38,8 @@ function install_pipeline_crd_version() {
   echo ">> Deploying Tekton Pipelines of Version $1"
   kubectl apply -f "https://github.com/tektoncd/pipeline/releases/download/$1/release.yaml" || fail_test "Build pipeline installation failed of Version $1"
 
-  if [ "${PIPELINE_FEATURE_GATE}" == "alpha" ]; then
+  # If the version to be installed is v0.40.x, we need to install resolvers.yaml separately.
+  if [[ "${1}" == v0.40.* ]]; then
```

Comment:
I had forgotten about this function... I guess it's used for update testing, which is not really working but good to keep it up to date as much as possible